src/main/scala/com/gu/datalakealerts/Athena.scala

```diff
@@ -13,6 +13,7 @@ object Athena {
 
   val client: AmazonAthena = AmazonAthenaClient
     .builder()
+    .withRegion("eu-west-1")
```

Comment:
Is that necessary or was it just to run it locally?

Reply:
just to run it locally! didn't know about `AWS_DEFAULT_REGION`. i'll remove these then.